### PR TITLE
docs: organizing designing resources

### PR DESCRIPTION
**Pull request by janchild**
Adding a new section called *Extensions* to the *Other resources* page to include different kits that we have to offer, including mobile, data viz. And in the future text toolbar, Walk me etc.

Add a line that some of these resources are only available to IBMers.

### CODE REVIEW

**Comment by laurenmrice** [comment]
Added text toolbar card in

File: src/pages/designing/design-resources/index.mdx [hunk 1]
```diff
@@ -97,14 +130,13 @@ For data visualization resources, go to the
 
 <Column colLg={4} colMd={4} noGutterSm>
   <ResourceCard
-    subTitle="Color contrast checker"
-    href="https://marijohannessen.github.io/color-contrast-checker/"
-    >
-
-![](images/color-contrast-icon.png)
-
+    subTitle="Text tooblar pattern library"
```
Comment (janchild) [inline src/pages/designing/design-resources/index.mdx:133]:
```suggestion
    subTitle="Text toolbar pattern library"
```

File: src/pages/designing/design-resources/index.mdx [hunk 2]
```diff
@@ -86,7 +84,42 @@ For data visualization resources, go to the
...
+    subTitle="Data visualization master sketch file"
+    href="https://github.com/carbon-design-system/carbon-design-kit/tree/master/Carbon%20Design%20Kit%20-%2010.10.0/data-visualization"
+    actionIcon="download">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Carbon mobile sketch kit"
```
Comment (janchild) [inline src/pages/designing/design-resources/index.mdx:115]:
```suggestion
    subTitle="Carbon mobile Sketch kit"
```

File: src/pages/designing/design-resources/index.mdx [hunk 3]
```diff
@@ -86,7 +84,42 @@ For data visualization resources, go to the
...
+</Column>
+</Row>
+
+## Extensions
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Data visualization master sketch file"
```
Comment (janchild) [inline src/pages/designing/design-resources/index.mdx:106]:
```suggestion
    subTitle="Data visualization master Sketch file"
```

**Review by andreancardona — APPROVED** [review]

**Review by jillianhowarth — APPROVED** [review]

**Pull request merged**